node/parallel-dev/src/chain_spec.rs

```diff
@@ -28,6 +28,8 @@ use vanilla_runtime::{
     WASM_BINARY,
 };
 
+pub const DEFAULT_LIQUIDATION_INCENTIVE_PCT: Ratio = Ratio::from_percent(110);
```

Comment:
Here we use Ratio to represent the fixed point number range from 0 to 1, and Rate to represent the fixed point number from 0 to a big number, better use Rate if change the percent from 90 to 110, also need to update the logic that uses this number in loans pallet.

Reply:
Thanks for the explanation. The code is now using `Rate` instead of `Ratio`